Corrected Parker weights references

GabrieleBelotti opened the pull request:
Corrected a description of the Parker filter to match the exact literature it's based upon. In particular, Parker's paper version available online still carries an error that was later amended, as seen in classic book "Principles of computerized tomographic imaging" by Kak and Slaney.

SimonRit commented:
I think that this class is solely based on Parker. I suggest to amend this PR by
* removing the reference to [Wesarg, Med Phys, 2002] (you could open an [issue](https://github.com/SimonRit/RTK/issues) to suggest the implementation of Wesarg)
* mentioning the correction to the denominator [here](https://github.com/SimonRit/RTK/blob/master/include/rtkParkerShortScanImageFilter.hxx#L196) (which fixes a typo in equation (12) of Parker)

GabrieleBelotti commented:
> I think that this class is solely based on Parker. I suggest to amend this PR by
> 
> * removing the reference to [Wesarg, Med Phys, 2002] (you could open an [issue](https://github.com/SimonRit/RTK/issues) to suggest the implementation of Wesarg)
> * mentioning the correction to the denominator [here](https://github.com/SimonRit/RTK/blob/master/include/rtkParkerShortScanImageFilter.hxx#L196) (which fixes a typo in equation (12) of Parker)

Agreed! Removed Wesarg reference and added a note on Parker's typo in header and source file at relevant line.

SimonRit commented:
Thanks. Can you squash the commits in one with a commit message such as "DOC: Correct Parker reference and its implementation"?
